pkgs/tools/system/kanata/default.nix

```diff
@@ -1,6 +1,8 @@
-{ lib
+{ stdenv
+, lib
 , rustPlatform
 , fetchFromGitHub
+, IOKit
```

Comment:
Consider including darwin instead. While inheriting specific packages was the thing to do historically, this will prevent moving the package to the `by-name` space in the future.

Reply:
> Consider including darwin instead.

What do you mean by that?  Is [this][1] the comment you refer to?

[1]: https://github.com/NixOS/nixpkgs/pull/281437/files#r1458023659

Reply:
Instead of inheriting `IOKit` itself in all-packages, add darwin to arguments and either add the whole `darwin.apple_sdk.frameworks.IOKit` to dependencies, or `let` assign the name like in https://github.com/NixOS/nixpkgs/blob/4e1582c0136f371f68c23074acf8ae22ddb14a0a/pkgs/by-name/cm/cmake/package.nix#L35

This allows the main package to be used without the extra IOKit option, so we'll be able to move it to the `by-name` area.